src/utils/format-phone-number.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Formats the given phone number string.
+ * @example '+91-1234567890' -> '+91-1234567890'
+ * @example '123' -> '123'
+ * @example '1234567' -> '123-4567'
+ * @example '1234567890' -> '(123)-456-7890'
+ * @example '1234567890123' -> '1234567890123'
```

Comment:
This example is actually different than the previous V-Mask format: the old code did not display any digits after 10. The new code displays them unformatted.